js/msp.js

```diff
@@ -407,9 +409,9 @@ var MSP = {
                 }
                 if (semver.gte(CONFIG.apiVersion, "1.10.0")) {
                     RC_tuning.RC_YAW_EXPO = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
-                    if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
+                    if (semver.gte(CONFIG.flightControllerVersion, "2.9.1")) {
```

Comment:
We should either go for all checks apiVersion or all checks flightControllerVersion in one file. Having a complete mess of both makes this a nightmare to debug.